addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/welcome/NAWelcomeCameraHandler.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.netatmo.internal.welcome;
+
+import static org.openhab.binding.netatmo.NetatmoBindingConstants.*;
+import static org.openhab.binding.netatmo.internal.ChannelTypeUtils.*;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.openhab.binding.netatmo.handler.NetatmoModuleHandler;
+
+import io.swagger.client.model.NAWelcomeCamera;
+
+/**
+ * {@link NAWelcomeCameraHandler} is the class used to handle the Welcome Camera Data
+ *
+ * @author Ing. Peter Weiss - Welcome camera implementation
+ *
+ */
+public class NAWelcomeCameraHandler extends NetatmoModuleHandler<NAWelcomeCamera> {
+    private final static String LIVE_PICTURE = "/live/snapshot_720.jpg";
```

Comment:
Non-preferred order of modifiers (It was like this in a template, which has been changed recently) correct order is `public static final` See also: http://stackoverflow.com/questions/16731240/what-is-a-reasonable-order-of-java-modifiers-abstract-final-public-static-e#16732059